airflow/requirements.txt

```diff
@@ -1,13 +1,15 @@
-/tmp/airflow_provider_weaviate-1.0.0-py3-none-any.whl
+/tmp/airflow_provider_weaviate-0.0.1-py3-none-any.whl
 apache-airflow-providers-github==2.3.1
 apache-airflow-providers-slack==7.3.2
 weaviate-client==3.22.0
 streamlit==1.25.0
-html2text
+html2text==2020.1.16
 pypandoc==1.11
 langchain==0.0.240
 markdownify
 stackapi==0.2.0
-protobuf==3.20.1
-google-cloud-firestore
-openai
\ No newline at end of file
+protobuf==3.20.2
+google-cloud-firestore==2.12.0
+openai==0.28.1
+bs4==0.0.1
```

Comment:
Looks like we're removing it in this pr. do we still need it?

Reply:
I think you will still need bs4, right?